app/src/common/shared/org/mozilla/vrbrowser/VRBrowserActivity.java

```diff
@@ -383,6 +397,102 @@ public void onRequestPermissionsResult(int requestCode, String[] permissions, in
         }
     }
 
+    //Settings
+    private void showSettingsWidget() {
```

Comment:
I'd prefer that we move all this code to the SettingsWidget class itself. We don't want VRBrowerActivity class become very big. You can access the WidgetManager from any widget to show/hide things.

(e.g. Ideally browser window or navigation bar just calls mSettingsWidget.show())